src/App.js

```diff
@@ -0,0 +1,109 @@
+import { Component } from 'react';
+import AppHeader from './AppHeader';
+import ItemsList from './ItemsList';
+import AppFooter from './AppFooter';
+
+class App extends Component {
+    constructor() {
+        super()
+        this.state = {
+            inputfield: '',
+            inputs: [],
+            saveButtonClassActive: false
+        };
+    }
+
+    componentDidMount() {
+        const storageItems = localStorage.getItem('items');
+        const parsedItems = JSON.parse(storageItems);
+        if (parsedItems) {
+            const items = [];
```

Comment:
can you please change this to reduce?

Reply:
Sure, I changed the `for...of` to `reduce`.

Reply:
well, probably you're right, `.map` is more appropriate.

the reason I suggest `.reduce`, because I thought that you're reducing `parsedItems` into `items`, but you actually remapping it, so proper solution would be `.map`

Reply:
Ok, I changed it to `map`.